docs/get-involved/join-the-drips-community.md

```diff
@@ -3,9 +3,9 @@ id: join-the-drips-community
 title: Join the Drips Community
 ---
 
-The Drips community is a part of the greater [Radicle community][rc].
+The Drips community is a part of the [Radicle community][rc].
 
-To get involved with the Drips community, you can find us in the #drips channel on the [Radicle Discord Server][rd].
+To get involved with Drips, come find us in the #drips and #drips-dev channels in the [Radicle Discord][rd].
```

Comment:
(Same here, clicking the links navigates out of the docs page)

Reply:
Fixed this